seacrowd/sea_datasets/palito/palito.py

```diff
@@ -0,0 +1,156 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import (DEFAULT_SEACROWD_VIEW_NAME,
+                                       DEFAULT_SOURCE_VIEW_NAME, Tasks)
+
+_DATASETNAME = "palito"
+_SOURCE_VIEW_NAME = DEFAULT_SOURCE_VIEW_NAME
+_UNIFIED_VIEW_NAME = DEFAULT_SEACROWD_VIEW_NAME
+
+_CITATION = """
+Shirley Dita, Rachel Edita Roxas, Paul Inventado. 2008. Building online corpora of Philippine languages. Proceedings of the 23rd Pacific Asia Conference on Language, Information and Computation. Available online: http://aclweb.org/anthology/Y/Y09/Y09-2024.pdf
+"""
```

Comment:
consider changing the `_CITATION` to BibTex Format, as usually LaTeX is generally used by Academia.
You can find the BibTex format from this:
<img width="1186" alt="image" src="https://github.com/SEACrowd/seacrowd-datahub/assets/69744460/bd8488af-833f-49d3-a5ca-d55d72c4956d">

```suggestion
_CITATION = """
@inproceedings{dita-etal-2009-building,
    title = "Building Online Corpora of {P}hilippine Languages",
    author = "Dita, Shirley N.  and
      Roxas, Rachel Edita O.  and
      Inventado, Paul",
    editor = "Kwong, Olivia",
    booktitle = "Proceedings of the 23rd Pacific Asia Conference on Language, Information and Computation, Volume 2",
    month = dec,
    year = "2009",
    address = "Hong Kong",
    publisher = "City University of Hong Kong",
    url = "https://aclanthology.org/Y09-2024",
    pages = "646--653",
}
"""
```

Reply:
Thanks!